src/Statement.php

```diff
@@ -251,6 +251,7 @@ public function verify($options = array()) {
         }
         else if (isset($header['x5c'])) {
             $cert = "-----BEGIN CERTIFICATE-----\r\n" . chunk_split($header['x5c'][0], 64, "\r\n") . "-----END CERTIFICATE-----\r\n";
+
```

Comment:
Not sure if this was here last time I reviewed; apologies for missing it if it was. 

It doesn't seem like I good idea to edit a file and bump the last modified date just to add a new line.